addons/godot-twicil/godot-twicil/helpers/tools.gd

```diff
@@ -0,0 +1,45 @@
+class_name HelperTools
+
+func __equals_string(str1, str2):
+    return str1 == str2
+
+func __equals_one_of_strings(str1, str_list):
+    return str1 in str_list
+
+func split_string(string, splitter, splits_count=0):
```

Comment:
Seems like this is already implemented in 3.x engine version. Remove if unnecessary.